micro-rdk/src/common/webrtc/api.rs

```diff
@@ -79,27 +84,23 @@ pub enum WebRtcError {
     #[error("cannot parse candidate")]
     CannotParseCandidate,
     #[error("Operation timeout")]
-    OperationTiemout,
+    OperationTimeout,
     #[error(transparent)]
     GrpcClientError(#[from] GrpcClientError),
 }
 
+pub(crate) type WebRtcSignaling = Either<AppSignaling, LocalSignaling>;
```

Comment:
This is the key change. We now have two different types that can represent a signaling request. One for app signaling, and one for local signaling.

I'm open to a better name for this type given how similar it is to `WebRtcSignalingChannel`. Maybe `WebRtcSignalingInvocation`?